storage/WindowAggregationOperationState.cpp

```diff
@@ -126,8 +121,8 @@ WindowAggregationOperationState* WindowAggregationOperationState::ReconstructFro
   const std::int64_t num_preceding = proto.num_preceding();
   const std::int64_t num_following = proto.num_following();
 
-  return new WindowAggregationOperationState(database.getRelationSchemaById(proto.relation_id()),
-                                             aggregate_function,
+  return new WindowAggregationOperationState(database.getRelationSchemaById(proto.input_relation_id()),
+                                             window_aggregate_function,
```

Comment:
FYI, we could directly use `&WindowAggregateFunctionFactory::ReconstructFromProto(proto.function())` for `window_aggregate_function`.